samples/repo-search/desktopApp/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+
+plugins {
+  alias(libs.plugins.kotlin.multiplatform)
+  alias(libs.plugins.jetbrains.compose)
+}
+
+kotlin {
+  jvm {
+    withJava()
+  }
+
+  sourceSets {
+    val jvmMain by getting {
+      dependencies {
+        implementation(compose.desktop.currentOs)
+        implementation(projects.samples.repoSearch.shared)
+        api(projects.pagingRuntimeComposeui)
+        implementation("org.jetbrains.kotlinx:kotlinx-coroutines-swing:1.6.0")
```

Comment:
The lack of this dependency might've been the source of your trouble with coroutines. Found it discussed in [the Compose JB release notes](https://github.com/JetBrains/compose-jb/releases/tag/v1.1.1).